### ISSUE TITLE
Preserving Top-Level Directories in Files List

### ISSUE THREAD

**Issue by AntumDeluge** [AntumDeluge/debreate#10]
The option to preserve top-level directories when adding a folder to the files list has been added: ce876fbde0a2d28f401bec0ea9c8db7a6dd9600b

Please test.

![treeopts](https://cloud.githubusercontent.com/assets/3631473/22158180/e431033a-deef-11e6-9a7d-ad141bba30b5.png)

**Comment by AntumDeluge** [comment]
DEB package for testing can be downloaded here: [v0.7.next15](../releases/tag/v0.7.next15)

**Comment by brnW** [comment]
It works if both "Preserve top-level directories" and "List files individually" are ticked.
For example, with this tree

usr
...├── bin
...│.......└── tesseract-gui.py
...└── share
............├── applications
............│.........└── tesseract-gui.desktop
............└── pixmaps
.......................└── tesseract-gui-icon.svg

Target = /

File = ....................................................................... Staged Target =
/usr/bin/tesseract-gui.py .......................................... /
/usr/share/applications/tesseract-gui.desktop ......... /
/usr/share/pixmaps/tesseract-gui-icon.svg .............. /

And after save project, quit and open project again, I've got this

File = .....................................................................  Staged Target =
tesseract-gui.py ...................................................... /usr/bin/
tesseract-gui.desktop ............................................. /usr/share/applications/
tesseract-gui-icon.svg ............................................ /usr/share/pixmaps/

Is it possible without save/quit/reload ?
Thanks

**Comment by AntumDeluge** [comment]
Yes, I think it is possible, & definitely should be the behavior. Thank you for pointing it out. I will get to work on it.
